free-3.md

```diff
@@ -56,13 +115,29 @@ spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
-
+I think the language aspect would take around 60 percent of the time since it 
+requires one to design a very simple syntax that the user could easily pick up.
+This probably means making the language similar to plain English, but it would 
+need to be organized in such a way that it is easy for the computer to parse. One
+would need to specify the number of and types of parameters that can be associated 
+with each task and also specify the way in which the code should be grouped and
+organized from the user end. The "systems" aspects would take the rest of the time,
+and most of the work would be in defining the algorithm for task prioritization and
+formatting the output nicely.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
-
```

Comment:
I think that of all your ideas, I have the clearest idea about how to implement this one! This also seems to best fit our discussions about a DSL.